package.json

```diff
@@ -50,6 +53,7 @@
     "oref0-meal": "./bin/oref0-meal.js",
     "oref0-mint-max-iob": "./bin/oref0-mint-max-iob.sh",
     "oref0-normalize-temps": "./bin/oref0-normalize-temps.js",
+    "oref0-ns-autotune": "./bin/oref0-ns-autotune.sh",
```

Comment:
Why not cal it `oref0-autotune`? The autotune will tune `oref0`. It will not tune anything to Nightscout.